From f341ac0cfaa624490df7f4922d97aa20a46359f5 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:19:42 -0400
Subject: [alsa] i2c/ak413x: Use SNDRV_DEV_CODEC for ak413x codec objects

Message-id: <140972823573-119-patchset-jkysela@redhat.com>
Patchwork-id: 90819
O-Subject: [RHEL 7.1 PATCH 118/442] ALSA: i2c/ak413x: Use SNDRV_DEV_CODEC for ak413x codec objects
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

... instead of SNDRV_DEV_LOWLEVEL.
No functional change at this point.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 483eb0627683b840d8f74fc0bb75be9132e76cd2)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/i2c/other/ak4113.c b/sound/i2c/other/ak4113.c
index e04e750..1a3a6fa 100644
--- a/sound/i2c/other/ak4113.c
+++ b/sound/i2c/other/ak4113.c
@@ -98,7 +98,7 @@ int snd_ak4113_create(struct snd_card *card, ak4113_read_t *read,
    AK4113_CINT | AK4113_STC);
  chip->rcs1 = reg_read(chip, AK4113_REG_RCS1);
  chip->rcs2 = reg_read(chip, AK4113_REG_RCS2);
- err = snd_device_new(card, SNDRV_DEV_LOWLEVEL, chip, &ops);
+ err = snd_device_new(card, SNDRV_DEV_CODEC, chip, &ops);
  if (err < 0)
   goto __fail;
 
diff --git a/sound/i2c/other/ak4114.c b/sound/i2c/other/ak4114.c
index 15ae025..c7f5633 100644
--- a/sound/i2c/other/ak4114.c
+++ b/sound/i2c/other/ak4114.c
@@ -111,7 +111,7 @@ int snd_ak4114_create(struct snd_card *card,
  chip->rcs0 = reg_read(chip, AK4114_REG_RCS0) & ~(AK4114_QINT | AK4114_CINT);
  chip->rcs1 = reg_read(chip, AK4114_REG_RCS1);
 
- if ((err = snd_device_new(card, SNDRV_DEV_LOWLEVEL, chip, &ops)) < 0)
+ if ((err = snd_device_new(card, SNDRV_DEV_CODEC, chip, &ops)) < 0)
   goto __fail;
 
  if (r_ak4114)
-- 
1.7.1